src/utils/button/theme.js

```diff
@@ -1,8 +1,14 @@
 import * as c from '../../colors';
-import {SIZE} from './constants';
-import {hexToRgba} from '../../utils';
+import {hexToRgba} from '../';
 
-const createBaseColorSkin = (color, bg, border, hoverColor, hoverBg, hoverBc, activeColor, activeBg, activeBc, disabledColor, disabledBg, disabledBc) => ({
+export const SIZE = {
```

Comment:
Need to rething whether the theme should be for all buttons, or maybe brak to smaller themes